### DISCUSSION TITLE
Hide Alerting Contact points and Notification policies from Viewer Role

### DISCUSSION THREAD

**Discussion by AnthonyCheung327** [grafana/grafana#62285] [Alerting]
Would like to explore the option on hiding Alerting -> contact points and notification policies from Viewer Role.  As a normal Dashboard viewers, he/she shouldnt allow to view the alertmanger settings such as matching label under notification policies from my point of view. 
Please kindly accept this feature request, thanks

**Comment by zachjacobs** [comment]
This would be a large value add for us as well!  Following along.

**Comment by zachjacobs** [comment]
Alerting groups that are part of folders are visible by team members that do not have access to that folder.  This seems like more of a bug to me.

For example, If a user is part of team1 and only has read access to team1_folder, when I go to browse dashboards, I can only see the "General" folder and the "team1_folder".  This is the desired result.  However, if I navigate to Alerting->groups and group by folder I can see alerts that belong to all folders in my organization.  This is undesired.

**Comment by guilhermebrgs** [comment]
Any answers on this? This is a considerable security flaw.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82584. Feel free to continue the discussion around this there. Thank you!
